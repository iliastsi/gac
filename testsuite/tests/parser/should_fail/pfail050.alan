utf8_chars() : proc
{
(* Ωστόσω δεν υπάρχει υποστήριξη για ελληνικά
 * μέσα σε chars (αφού πρέπει να καταλαμβάνουν
 * μονάχα 1 byte
 *)
  writeString("Ηλίας\ν");
  writeByte('φ');
}
