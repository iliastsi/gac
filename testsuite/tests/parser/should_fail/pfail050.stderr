pfail050.alan:8:13: Error: Parse error on input `''
	Cannot parse char `''

pfail050.alan:8:14: Error: Parse error on input `φ'
	Cannot parse char `φ'

pfail050.alan:8:15: Error: Parse error on input `''
	Cannot parse char `''

==================== Parser ====================
utf8_chars() : proc
{ -- utf8_chars
  writeString("Ηλίαςν");
  writeByte();
} -- utf8_chars


